.changeset/free-pears-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugins': patch
```

Comment:
```suggestion
'@backstage-community/plugin-scaffolder-backend-module-jenkins': patch
```

Not sure what happened here bit this isn't the correct package. My suggestion has the correction.